python/Dockerfile

```diff
@@ -20,6 +21,6 @@ LABEL summary="$SUMMARY" \
     version="$tag_version" \
     maintainer=""
 
-RUN pip install --no-cache-dir aries-cloudagent==${agent_version}
+RUN pip install --no-cache-dir aries-cloudagent${acapy_reqs}==${agent_version}
```

Comment:
Wouldn't this need to be something like `aries-cloudagent==${agent_version},${acapy_reqs}` to install all of the packages, and particularly `aries-cloudagent` at the desired version?

Reply:
What I did appears to work in all the cases I tried.  I don't know pip well, so defer to others if there is a better way.

I would like to push a new image if people are happy with this.

Reply:
@andrewwhitehead any fedback? With the current string interpolation/formatting I believe the version would be applied - if anything -  to the last package listed in the `acapy_reqs` parameter rather than `aries-cloudagent`. Unless I am missing something the final command doesn't look right to me:
```
RUN pip install --no-cache-dir aries-cloudagentaskar,bbs==0.7.1
```

Reply:
I think there are meant to be square brackets in the requirements string, ie. `RUN pip install --no-cache-dir aries-cloudagent[askar,bbs]==0.7.1`

Reply:
Derp, that makes more sense now - I was missing the fact the brackets get carried across 🤦🏻‍♂️